docs/guides/integration-mobile-client-server.md

```diff
@@ -307,4 +307,48 @@ UID2Manager.shared.automaticRefreshEnabled = false
 
 ## Optional: UID2 Prebid Mobile SDK Integration
 
-<PrebidMobileSDK />
\ No newline at end of file
+<PrebidMobileSDK />
+
+## Error Response States
+
+[**GWH_IB_01 Are these states equally applicable to client-side and server-side?**]
+
+[**GWH_IB_02 Are these states equally applicable to Android and iOS, and identical? (if yes on both of these questions I will make this into a snippet when we've ironed out the wording)**]
```

Comment:
The error states are present on both Android and iOS.

/cc @dcaunt

Reply:
Perfect, thx. Removed the query.